docs/examples/synctime.c

```diff
@@ -189,10 +189,10 @@ size_t SyncTime_CURL_WriteHeader(void *ptr, size_t size, size_t nmemb,
 void SyncTime_CURL_Init(CURL *curl, char *proxy_port,
                         char *proxy_user_password)
 {
-  if(strlen(proxy_port) > 0)
+  if(*proxy_port)
     curl_easy_setopt(curl, CURLOPT_PROXY, proxy_port);
 
-  if(strlen(proxy_user_password) > 0)
+  if(*proxy_user_password)
```

Comment:
I think there is a case to be made for documentation to be kept as simple/readable as possible, so I vote for keeping `strlen()`.

Reply:
Done